internalsigner.go

```diff
@@ -54,14 +60,19 @@ func (is *InternalSigner) SignAuto(unsignedTx *Tx) (*Tx, error) {
 
 	for i, in := range unsignedTx.Inputs {
 
+		// todo: error is not being checked
```

Comment:
hmm i remember having this here because i wanted to do something with signing non-p2pkh inputs...